<p>I did nothing special, just restarted after a cold backup as I already did many times, but this time I got this error, and Alfresco is not starting:</p>  <pre><code>Caused by: org.postgresql.util.PSQLException:  ERROR: duplicate key value violates unique constraint "parent_node_id" Dettaglio: Key (parent_node_id, type_qname_id, child_node_name_crc, child_node_name)=(24947, 35, 3152099996, script.js) already exists. </code></pre>  <p>As a quick solution I went into postgresql tables and changed name of parent node to <code>script.js</code> to <code>new.script.js</code> and Alfresco started fine, but I soon noticed few things in the directory where script.js belong, which is the Alfresco Script directory in Data Dictionary:</p>  <ol> <li>In the directory I end up with two file named <code>scripts.js</code> </li> <li>Some of the file in the directory, but not all, were of an outdated  version, and created by System user.</li> </ol>  <p>Things I already tried:</p>  <ol> <li>restore the latest backup, nothing to do</li> <li>Full solr reindex, same error</li> </ol>  <p>This is my personal repository, I of course have a series of back up, so I can do any test I want. Please give advice for restoring my repository.</p>  <p>Few details about the installation:</p>  <ol> <li>Alfresco community 4.2.f (share + explorer)</li> <li>Single installation, no cluster or replication</li> <li>No external database, all in the same folder</li> <li>Default installation with some custom amps and Quick Start module</li> <li>The previous stop back up and restart went fine</li> </ol>